src/lib/dux/sdk/actionTypes.ts

```diff
@@ -0,0 +1,12 @@
+import { ObjectValues } from "../../../utils/typeHelpers/objectValues";
+
+const actionTypes = {
```

Comment:
nit, call this `ACTION_TYPES`

Reply:
Lets make that our convention ~

Reply:
Ok ok

Reply:
I think It's good to change all in the different PR